### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by dtau00** [appy-one/acebase#123] [Q&A]
tried following the documentation but couldn't get the syntax right for either the rules file or programmatically.  in this example im trying to create an array of messages (specifically using an array and not a collection)

  await db.schema.set('users/$uid',{
      journal: {
          id: 'string',
          messages:{
            uid: "string",
            msg: "string"
          }[]
      }
  })

what's the right way to do this?  also tried it as a string
messages:'{
  uid: "string",
  msg: "string"
}[]'

**Comment by dtau00** [comment]
can ignore, think ive figured it out

**Reply by appy-one** [reply]
Please share your solution for other readers 👍🏼
